underfs/oss/pom.xml

```diff
@@ -43,6 +43,10 @@
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpcore</artifactId>
     </dependency>
+    <dependency>
```

Comment:
please keep the external dependencies sorted lexicographically ... so move this above `httpcore`